Competition2017/src/competition/subsystems/agitator/AgitatorSubsystem.java

```diff
@@ -41,30 +59,62 @@ public AgitatorSubsystem(
         this.agitatorMotor.createTelemetryProperties(side.toString(), propManager);
     }
 
-    public void setAgitatorPower(double power) {
+    protected void setAgitatorPowerRaw(double power) {
         agitatorMotor.ensureTalonControlMode(TalonControlMode.PercentVbus);
         agitatorMotor.set(power);
     }
+    
+    public void setAgitatorPower(double power) {
+        setAgitatorPowerRaw(power);
+        unjamEnd = -1;
+        isIntaking = false;
```

Comment:
ah nvm, I see how things are layered now, I think it's risky for this to be public since someone else could mess up the state machine by calling it directly?